backend/config/routes.rb

```diff
@@ -2,6 +2,7 @@
   namespace :admin do
     get '/search/users', to: "search#users", as: :search_users
     get '/search/products', to: "search#products", as: :search_products
+    post "/quick-switch", to: "quick_switch#find_object", as: :quick_switch
```

Comment:
I think `get` would be more appropriate then `post`.

Reply:
I also think `GET` is better here. This returns a redirect but it's still data retrieving (more like a question then the request to change something).